Bluepill-cli/BPInstanceTests/BPTestHelper.m

```diff
@@ -41,6 +41,11 @@ + (NSString *)sampleAppFatalTestsBundlePath {
 + (NSString *)resourceFolderPath {
     return [[NSBundle bundleForClass:[self class]] resourcePath];
 }
+
++ (NSString *)bpExecutablePath {
```

Comment:
Should we also use this in `BPRunner.m:BPRunnerForApp`? There we do:

```
        NSString *argv0 = [[[NSProcessInfo processInfo] arguments] objectAtIndex:0];
        NSString *bp = [[argv0 stringByDeletingLastPathComponent] stringByAppendingPathComponent:@"bp"];
```

Seems like we should only have one way of getting the path of the `bp` executable.